packages/css/package.json

```diff
@@ -63,7 +69,13 @@
     ],
     "exports": {
       "extra": {
-        "./macro": "./macro.js"
+        "./macro": {
+          "types": {
+            "import": "./macro.d.mts",
+            "default": "./macro.d.ts"
```

Comment:
```suggestion
```

Reply:
hm, i wasn't sure if this would "fall through" to the other default... does it? 🤔

Reply:
It does (but that's correct behaviour, not like TS's bug)